src/systems/eu/specs/components/link/package.json

```diff
@@ -21,6 +21,6 @@
     "design-system"
   ],
   "devDependencies": {
-    "@ecl/ec-style-typography-paragraph": "^2.0.0-alpha.1"
+    "@ecl/eu-style-typography": "^2.0.0-alpha.1"
```

Comment:
Don't forget to update `src/systems/eu/specs/components/link/docs/code.mdx` (same for EC)